test/e2e/tests/network/add-custom-network.spec.js

```diff
@@ -66,7 +66,7 @@ const MOCK_CHAINLIST_RESPONSE = [
 const selectors = {
   accountOptionsMenuButton: '[data-testid="account-options-menu-button"]',
   informationSymbol: '[data-testid="info-tooltip"]',
-  settingsOption: { text: 'Settings', tag: 'div' },
+  settingsOption: '[data-testid="global-menu-settings"]',
```

Comment:
I think this wouldn't be needed, but I guess it's fine

Reply:
Actually, this locator strategy was sufficient, and with only this change, the CI has passed. However, I added an additional safety net to check for MMI and wait for the 'Portfolio Dashboard'.